NewSpring/Rock
# Update Locations Pages to Use a Persisted Dataset

## DESCRIPTION

### What does this PR do, or why is it needed?
This PR: 
* Updates location lava templates to pull their data from a persisted dataset.
* Changes the "Get Directions" buttons to be regular links. 

### How do I test this PR?
* Copy the persisted dataset from https://rich-rock.newspring.cc/page/2396?PersistedDatasetId=1 and create or update the Campus dataset in your database.
* Pull this code and then run your site.
* Look at the locations pages ( `/locations`, `/locations/{campus}`, `/fuse/locations`, and `/fuse/locations/{campus}` and make sure that everything loads and looks correct.

## TODO


## REVIEW


> The purpose of PR Review is to _improve the quality of the software._
